api/views/api.py

```diff
@@ -11,15 +12,23 @@ def get_data(request):
         if request is None:
             raise Exception("Houston, we have a problem.")
 
-        search = request.GET.get("search")
+        # search = request.GET.get("search")
         page = request.GET.get("page")
         per_page = request.GET.get("per_page")
```

Comment:
I would make these default to something. If page isn't there, give me the first page, if per page isnt there default to 10 or 20

```
page = request.GET.get("page", 1)
per_page = request.GET.get("per_page", 10)
```